homeassistant/components/ddwrt/device_tracker.py

```diff
@@ -145,6 +196,61 @@ def get_ddwrt_data(self, url):
             return
         _LOGGER.error("Invalid response from DD-WRT: %s", response)
 
+    def get_extra_attributes(self, device):
+        """Return the extra attributes of the devices wireless connections."""
+        if not self.monitored_conditions:
+            return {}
+
+        url = '{}://{}/Status_Wireless.live.asp'.format(
```

Comment:
Single call to get all data... filtering with monitorred conditions should not be done as per [ADR 0003](https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md)